webapp/views/App/views/Users/UserEdit/UserEdit.js

```diff
@@ -38,79 +39,95 @@ const UserEdit = () => {
   const i18n = useI18n()
   const validation = User.getValidation(userToUpdate)
 
+  const titleItems = ['mr', 'ms', 'preferNotToSay']
```

Comment:
maybe this key should go into the `core/user/_user/userProps` like propTitleKeys